src/compiler/checker.ts

```diff
@@ -7652,6 +7636,33 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
             }
         }
 
+        function symbolToParameterDeclaration(parameterSymbol: Symbol, context: NodeBuilderContext, preserveModifierFlags?: boolean, privateSymbolVisitor?: (s: Symbol) => void, bundledImports?: boolean): ParameterDeclaration {

```

Comment:
This got moved; is there any chance you could move it back to where it was for a better diff?

Reply:
I didn't move it but I prepended it with 2 functions that I extracted from it. Git decided that the other parts were not moved, I could juggle this around to force a different diff but logically it makes sense to me that `parameterToParameterDeclarationName` comes before `symbolToParameterDeclaration`

Reply:
Funny, I often put my helpers below the function that uses it, so you read the high level thing first... 😄

Reply:
I guess it differed enough that git forgot, very annoying 🙁

Reply:
> Funny, I often put my helpers below the function that uses it, so you read the high level thing first...

It's weird, I'm okay with this but on the other hand stuff like

```ts
interface I {
    foo: Foo;
    bar: Bar;
}
interface Foo {
    /* ... */
}
interface Bar {
    /* ... */
}
```

annoys me to no end.

Reply:
Reviewing the diff, the change is definitely if you move the functions around...